components/treeview/index.css

```diff
@@ -19,6 +19,9 @@ governing permissions and limitations under the License.
   padding: 0;
   user-select: none;
   outline: none;
+
+  height: 500px;
+  width: 200px;
```

Comment:
These changes come from RSP V3 repo.

Reply:
do these need to be hard-coded or are they coming from some DNA var?
Or could those be built with calc()

Reply:
These need to be removed completely

Reply:
They look removed now.